sites/funmooc/src/backend/base/tests/test_views.py

```diff
@@ -49,13 +93,21 @@ def test_views_redirect_edx_courses_fallback_search_page(self):
         nor the organization page can be found.
         """
         PageFactory(
-            reverse_id="search",
+            reverse_id="courses",
             template="search/search.html",
             title__title="Recherche",
             title__language="fr",
             should_publish=True,
         )
 
+        PageFactory(
+            reverse_id="sorbonne",
+            template="richie/single_column.html",
+            title__title="Sorbonne",
+            title__language="fr",
+            should_publish=True,
+        )
+
```

Comment:
I don't get why this factory is needed either…

Reply:
Just to ensure that if a page has a reverse_id matching organization argument but it is not an organization page, we did not return this page.

Maybe I have to explain this in a comment.

Reply:
ok I see, thanks :)